src/App.jsx

```diff
@@ -808,7 +808,7 @@ const App = () => {
 
     return (
         <div className="App">
-            <div className="top-nav">
+             <div className="top-nav">
```

Comment:
You could abstract this "top-nav" part to its own component to separate from the whole "App".